backend/covexit/account/models.py

```diff
@@ -1,7 +1,43 @@
+import random
+import string
+
 from django.contrib.auth.models import User
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
 from django.core.exceptions import ValidationError
+from django.core.mail import send_mail
+from django.conf import settings
+
+VERIFICATION_MESSAGE = """Thank you for signing up for Covexit!
+
+To activate your account, please visit this link:
+{}
+"""
+
+VERIFICATION_KEY_LENGTH = 30
+# Note: this will have to be set up on the frontend:
+VERIFICATION_URL = 'https://covexit.de/verify/'
```

Comment:
Can we get the host name from somewhere so it also works locally or is that not a good a idea?